Framework/Algorithms/src/FFT.cpp

```diff
@@ -145,10 +154,8 @@ void FFT::exec() {
     }
   }
 
-  // centerShift == true means that the zero on the x axis is assumed to be in
-  // the data centre
-  // at point with index i = ySize/2. If shift == false the zero is at i = 0
-  const bool centerShift = true;
+  // Zero on the x axis is assumed to be in the centre,
+  // at point with index i = ySize/2.
 
```

Comment:
I never understood why we have `centerShift = true`. This causes the Fourier transform of:

![signal](https://cloud.githubusercontent.com/assets/8956985/14148455/70dd9060-f697-11e5-835d-06418345f246.png)

to be:

![fftsignal](https://cloud.githubusercontent.com/assets/8956985/14148461/7505ab82-f697-11e5-87b5-1288232f546c.png)

which does not seem right, as my data start at `x=0` and I would expect a positive peak at `w = w0`, being `w0` the frequency of the input signal.

I see why you've removed `centerShift` (it is set to `True` and never changes its value), I am just wondering if we should keep it to make it clear that for some reason we are considering that the x axis is assumed to be in the data center. This way, if at some point we want to set it to `false` (or maybe even add another input property to specify where our center should be) it should be just a matter of setting `const bool centerShift=false;`.
What do you think?

In any case your new property `AutoShift` fixes this behavior, which is good.